acceptance/tests/aix/aix_package_provider.rb

```diff
@@ -22,6 +18,11 @@ def assert_package_version(package, expected_version)
 version1 = '1.7.10.4'
 version2 = '1.8.6.4'
 
+teardown do
+  on hosts, "rm -rf #{dir}"
+  on hosts, puppet('resource', 'package', "'#{package}' ensure=absent")
```

Comment:
This gets shoved through the shell on the SUT, so the quotes will get eaten at that level. Redundant here since the package name doesn't contain any special characters, but probably not the worst practice.